File: DSA/Shubham_590016506/Day_16_Question_2.c
#include <stdio.h>

int countOccurrences(int arr[], int size, int target) {
    int count = 0;
    for (int i = 0; i < size; i++) {
        if (arr[i] == target)
            count++;
    }
    return count;
}

int main() {
    int arr[] = {1, 2, 3, 2, 2, 4};
    int size = 6;
    int target = 2;

    int count = countOccurrences(arr, size, target);
    printf("%d appears %d times\n", target, count);  // Output: 3
    return 0;
}

File: DSA/Pranvkumar_590011587/Day16_Question2.py
def count_occurrences(arr, target):
# Counts how many times the target appears in the array.
    count = 0
    for val in arr:
        if val == target:
            count += 1
    return count


# Example usage:
if __name__ == "__main__":
    # Count Occurrences Examples
    print(count_occurrences([1, 2, 3, 2, 2, 4], 2))         # Output: 3
    print(count_occurrences([5, 1, 5, 5, 3, 5, 7], 5))      # Output: 4
    print(count_occurrences([10, 20, 30, 10, 40], 10))      # Output: 2